### ISSUE TITLE
Aqueous humor tap: viral DNA positive

### ISSUE THREAD

**Issue by MickeySegal** [obophenotype/human-phenotype-ontology#10214]
**Preferred term label:**
Aqueous humor tap: viral DNA positive

One might create separate findings for CMV, HSV, and VZV but my guess is that the testing is done for all 3.

**Synonyms**

**Definition (free text, please give PubMed ID)**
This might seem like a research-only test, but https://www.ncbi.nlm.nih.gov/books/NBK576412/ says "Essential investigation in characterizing PSS is PCR analysis of aqueous humor tap to assess viral DNA. The common viruses assessed are CMV, HSV, and VZV."

**Parent term (use hpo.jax.org/app)**

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
Posner-Schlossman Syndrome

**Your nano-attribution (ORCID)**
